net/dns/resolved.go

```diff
@@ -332,6 +333,10 @@ func (m *resolvedManager) GetBaseConfig() (OSConfig, error) {
 	return OSConfig{}, ErrGetBaseConfigNotSupported
 }
 
+func (m *resolvedManager) GetExitNodeForwardResolver() ([]dnstype.Resolver, error) {
+	return []dnstype.Resolver{{Addr: "127.0.0.53:53"}}, nil
```

Comment:
Here's hoping no-one ever sets `DNSStubListener=no` in resolved.conf.

Reply:
That case will be fixed in a follow-up CL that also makes exit node DNS proxy work on Windows: using gethostbyname etc (via Go net package DNS lookups).

Reply:
FWIW: nobody runs resolved and disables the stub listener. It'd break too much stuff.